### DISCUSSION TITLE
Legion 7 Slim support?

### DISCUSSION THREAD

**Discussion by hinchyman** [4JX/L5P-Keyboard-RGB#57] [Ideas]
Any chance this could be ported to support the legion 7 slim? This model is having a lot of the same issues

**Comment by 4JX** [comment]
> Any chance this could be ported to support the legion 7 slim? This model is having a lot of the same issues

Unless I missed something the 7 Slim has the same corsair based keyboard as the 7 and therefore https://github.com/4JX/L5P-Keyboard-RGB#how-about-x-model applies sadly.
